node_modules/.yarn-integrity

```diff
@@ -0,0 +1,12 @@
+{
+  "systemParams": "darwin-arm64-83",
+  "modulesFolders": [
+    "node_modules"
+  ],
+  "flags": [],
+  "linkedModules": [],
+  "topLevelPatterns": [],
+  "lockfileEntries": {},
+  "files": [],
+  "artifacts": {}
+}
```

Comment:
🔧 If you'd like, you can remove this folder along with `package-lock.json` and `yarn.lock` - not sure how these ended up here but they shouldn't affect anything.